aea/cli/remove.py

```diff
@@ -396,8 +408,24 @@ def _remove_from_config(self) -> None:
             "Removing the {} from {}".format(self.item_type, DEFAULT_AEA_CONFIG_FILE)
         )
         self.agent_items.remove(current_item)
+        self.agent_config.component_configurations.pop(
+            ComponentId(self.item_type, current_item), None
+        )
+        self._dump_agent_config()
+
+    def remove_unused_component_configurations(self) -> None:
+        """Remove all component configurations for items not registered and dump agent config."""
+        registered_components = self.agent_config.package_dependencies
```

Comment:
call remove_unused_component_configurations

Reply:
sorry it's outdated and not used anymore.